WordPress/Classes/Services/BloggingPrompts/BloggingPromptRemoteObject.swift

```diff
@@ -0,0 +1,64 @@
+/// Encapsulates a single blogging prompt object from the v3 API.
+struct BloggingPromptRemoteObject {
+    var promptID: Int
+    var text: String
+    var attribution: String
+    var date: Date
+    var answered: Bool
+    var answeredUsersCount: Int
+    var answeredUserAvatarURLs: [URL]
+    var answeredLink: URL?
+    var answeredLinkText: String
```

Comment:
Do all of these properties need to be mutable? I don't see any being modified in the diff.

Reply:
Good catch! This is addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/21952/commits/2c9b009ebabcecc25a6bef3b1d49e51aa3027bd2. 🙂